lib/assetcompute.js

```diff
@@ -117,7 +121,11 @@ class AssetCompute {
         });
         if (!response.ok) {
             const msg = await response.text();
-            throw Error(`Unable to invoke /unregister: ${response.status}: ${msg} (details: ${JSON.stringify(response)})`);
+            const error = new Error(`Unable to invoke /unregister: ${response.status}: ${msg} (details: ${JSON.stringify(response)})`);
+            if (response.status === 429) {
+                error.retryAfter = parseInt(response.headers.get('retry-after')); 
+            }
+            throw error;
```

Comment:
the response headers were not sent back to the client since `!response.ok` was sent back as an error, so I added the `retry-after` header to the error.

An improvement would be to create a custom error class for these 429 errors